core/src/main/scala/org/apache/spark/scheduler/SchedulerBackend.scala

```diff
@@ -73,7 +75,12 @@ private[spark] trait SchedulerBackend {
    * Executors tab for the driver.
    * @return Map containing the log names and their respective URLs
    */
-  def getDriverLogUrls: Option[Map[String, String]] = None
+  def getDriverLogUrls: Option[Map[String, String]] = {
```

Comment:
Shall we explicitly mention the target resource managers instead of `Support setting driver log url using env vars on other resource managers` because `YarnClusterSchedulerBackend` will not use this implementation?

https://github.com/apache/spark/blob/b9998cf1bf3501936bdf7c0d2a8cc3ddf0115a06/resource-managers/yarn/src/main/scala/org/apache/spark/scheduler/cluster/YarnClusterSchedulerBackend.scala#L38-L40

Reply:
It makes sense if we want to keep YARN as-is, another direction is to let Yarn support it then it works on all resource managers.

The pseudo-code would like
```
override def getDriverLogUrls: Option[Map[String, String]] = { 
   YarnContainerInfoHelper.getLogUrls(sc.hadoopConfiguration, container = None) ++
     super.getDriverAttributes.getOrElse(Map.empty))
 } 
```

Reply:
it was mentioned in SPARK-26311